extern/storage-sealing/precommit_policy.go

```diff
@@ -60,7 +62,7 @@ func (p *BasicPreCommitPolicy) Expiration(ctx context.Context, ps ...Piece) (abi
 		return 0, err
 	}
 
-	var end *abi.ChainEpoch
+	var hardLimit *abi.ChainEpoch
```

Comment:
`hardLimit` sounds like this is a 'hard max', when this is minimum

Reply:
yessir.